docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -14,6 +14,21 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 * Installing CSI as part of VSphere cluster creation was deprecated in version `v0.16.0` and has been removed in `v0.17.0`. Please refer to the [deprecation section]({{< relref "../getting-started/vsphere/vsphere-getstarted/#vsphere-csi-driver-deprecation" >}}) in the vSphere provider documentation for more information.
 {{% /alert %}}
 
+## [v0.17.0]()
+
+### Adds
+
+### Upgrades
+
+### Fixes
+
+### Deprecates
+
+### Removes
+
+- Installing CSI as part of VSphere cluster creation.
```

Comment:
We need a docs page showing users 1) how to upgrade from v0.16 to v0.17 transitioning from EKS-A managed to self-managed vSphere CSI 2) how to self-manage vSphere CSI. That page needs to be linked here.